src/main/java/org/ehcache/sizeof/SizeOfEngine.java

```diff
@@ -0,0 +1,34 @@
+/**
+ *  Copyright Terracotta, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.ehcache.sizeof;
+
+/**
+ * SizeOf engines are used to calculate the size of elements stored in poolable stores.
```

Comment:
The javadoc is off, there are no elements nor poolable stores in the context of the sizeof engine.